### PR TITLE
IDE-3595 update liferay maven workspace version to latest

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
Build Failed.
 No test results found.
Test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 5

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/5/checker.html for detailed testing results

File: maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public void initializeDefaultPreferences() {
 
 		defaultPrefs.put(
 			LiferayMavenCore.PREF_ARCHETYPE_PROJECT_TEMPLATE_WORKSPACE,
-			"com.liferay:com.liferay.project.templates.workspace:1.0.4");
+			"com.liferay:com.liferay.project.templates.workspace:1.0.7");
```
Comment (gamerson) [inline maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java]: this should be done via blade now and we should no longer use it from the archetype.  @JoyeLuo  can you add support to the `blade init` command to use maven?  then we can use blade for creating a maven workspace.  Thanks!

Comment (gamerson) [reply maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java]: Then we can remove this template all together.

Comment (gamerson) [reply maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java]: cc @jtydhr88

Comment (jtydhr88) [reply maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java]: hey @gamerson can you merge it firstly as It causes some bugs as the old template use old blade.support lib, I will ask @JoyeLuo work on the Maven support today. thanks

**Comment by yanan** [comment]
Build Failed.
 No test results found.
Test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 1

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/1/checker.html for detailed testing results

**Comment by yanan** [comment]
Test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 1

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/1/checker.html for detailed testing results

**Comment by yanan** [comment]
Build Failed.
 7 tests run, 0 skipped, 7 failed.
SDK test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 1

**Comment by yanan** [comment]
SDK test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 1

**Comment by yanan** [comment]
Build Failed.
 9 tests run, 0 skipped, 9 failed.
Hook test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 2

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/2/checker.html for detailed testing results

**Comment by yanan** [comment]
Hook test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 2

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/2/checker.html for detailed testing results

**Comment by yanan** [comment]
Build Failed.
 8 tests run, 0 skipped, 8 failed.
Layout Template test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 2

**Comment by yanan** [comment]
Layout Template test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 2

**Comment by yanan** [comment]
Build Failed.
 18 tests run, 0 skipped, 18 failed.
Portlet test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 3

**Comment by yanan** [comment]
Portlet test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 3

**Comment by yanan** [comment]
Build Failed.
 13 tests run, 1 skipped, 12 failed.
Server test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 3

**Comment by yanan** [comment]
Server test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 3

**Comment by yanan** [comment]
Build Failed.
 25 tests run, 2 skipped, 23 failed.
Theme test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 3

**Comment by yanan** [comment]
Theme test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 3

**Comment by yanan** [comment]
Build Failed.
 18 tests run, 0 skipped, 18 failed.
Portlet test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 7

**Comment by yanan** [comment]
Portlet test Failed on Jenkins http://ide-build-1:9090/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 7

**Comment by yanan** [comment]
jenkins retest

**Comment by yanan** [comment]
Build Failed.
Test Failed on Jenkins http://ide-build-1:9090/job/Multiple-Job-Test/ on this commit 8819265bb1c01a35271e6028063c206f2d2f0673 on build 17

**Pull request closed without merging**